SuiteCRM/SuiteCRM
# Fix #9702  - PHP Warning on exec() in include/utils.php

## Description




Fixes #9702 warnings polluting logs when run on shared hosting with system shell commands disabled.

## Motivation and Context

Also asks user to install the required linux/unix package `php-posix` if not detected.

## How To Test This

Try to run without package `php-posix` should show message saying please install it.

## Types of changes


### Final checklist
